### DISCUSSION TITLE
R9M LITE PRO V1.5 does not have pads for STLink, how to flash?

### DISCUSSION THREAD

**Discussion by Alt-Pavlo-Delete** [ExpressLRS/ExpressLRS#2401] [Q&A]
Hello everybody. I bought R9M LITE PRO to flash it with ELRS, i had earlier usual R9 Lite and there were no problems, but this time i received module without SCK, SIO pads to flash it with STLink. Text on the main board says R9M LitePro V1.5, manufacturing year 2021. 
How can i flash ELRS in this case?

![image](https://github.com/ExpressLRS/ExpressLRS/assets/82888480/9b8e79ff-3490-4438-9ebc-9be50e2c3186)

**Comment by cvetaevvitaliy** [comment]
![267041588-9b8e79ff-3490-4438-9ebc-9be50e2c3186](https://github.com/ExpressLRS/ExpressLRS/assets/26421560/cfaaae9b-dbc9-4ac6-9bba-8221e00bc406)

It wouldn't hurt if you took pictures of both sides of the PCB  :)
